src/blob/core.rs

```diff
@@ -573,8 +586,19 @@ impl RawRecords {
             })?;
         self.current_offset += self.record_header_size;
         self.current_offset += header.meta_size();
+        let data =
+        if read_data {
```

Comment:
Please add indentation so it can be seen that this is part of the value assignment to `data`